recipe/meta.yaml

```diff
@@ -1,21 +1,25 @@
 {% set name = "soqt" %}
 {% set version = "1.6.0" %}
-{% set sha256 = "9f535af59f07c907022815679681bd345c9dec2f408c74833f6b1b24bca40e1f" %}
 
 package:
     name: {{ name }}
     version: {{ version }}
 
 source:
-    - url: https://github.com/coin3d/soqt/releases/download/SoQt-{{ version }}/soqt-{{ version }}-src.tar.gz
-      sha256: {{ sha256 }}
+    - url: https://github.com/coin3d/soqt/archive/ab0595fe864470451ca303afb9b49c796daf2c00.tar.gz
```

Comment:
this is very strange, you are building an unversioned version of soqt. why is this?

Reply:
Yes qt6 support is not in the last available version

Reply:
is your goal to publish this unreleased version in conda-forge?

Reply:
I will ask if it is possible to publish a new version.